index.html

```diff
@@ -9,6 +9,7 @@
 	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 	<link rel="stylesheet" href="core/css/uikit.owncloud.css" />
 	<link rel="stylesheet" href="core/css/uikit.core.css" />
+	<link rel="stylesheet" href="../node_modules/vuetify/dist/vuetify.min.css" />
```

Comment:
should be in code bundle - right?

Reply:
`<link rel="stylesheet" href="core/css/core.css" />`
bundled everything now